scripts/add-software.mjs

```diff
@@ -142,17 +143,77 @@ async function addDesign() {
 async function addPlugin() {
   console.log(`
 
-  ${chalk.bold.yellow('🙈 Oh no; You called our bluf!')}
-  ${chalk.gray('≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡')}
-
-  Adding plugins is not (yet) implemented 😬
+  ${chalk.bold.yellow('👕 Add a new plugin')}
+  ${chalk.gray('≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡≡')}
 
-  Sorry 🤥
+  We're going to add a new plugin to this repository. That's awesome 🎉
+  Let's start by picking the name for this plugin 🏷️
+  Try to keep it to one word that explains what the plugin does e.g. ${chalk.green(
+    'flip'
+  )}, ${chalk.green('mirror')},
+   ${chalk.green('round')}.
 
 `)
+
+  const { name } = await prompts({
+    type: 'text',
+    name: 'name',
+    message: 'What name would you like the plugin to have? ([a-z] only)',
+    validate: validatePluginName,
+  })
+
+  if (name) {
+    console.log('\n' + `  Alright, let's add ${chalk.green(name)} to plugins 🪄`)
+    createPlugin(name)
+    execSync('npm run reconfigure')
+    console.log(`  All done 🎉`)
+
+    try {
+      console.log(`
+
+  ${chalk.bold.yellow('✨ Summary')}
+  ${chalk.gray('≡≡≡≡≡≡≡≡≡≡')}
+
+  👉  We've created your plugin skeleton at ${chalk.green('plugins/plugin-' + name)}
+  👉  We've configured the packages via the ${chalk.green('pacakge.json')} file
+  👉  We've added ${chalk.green('plugins/plugin-' + name)} to the lab
```

Comment:
And this actually has to go. It's true of designs, but not plugins

Reply:
Another typo from design section, updated both

Reply:
Wait so I am removing that entire section now

Reply:
no, just line 179

Reply:
Done that then